.travis.yml

```diff
@@ -0,0 +1,8 @@
+---
+# This tests the modulesync configuration _itself_.
+# The config used for modules is at moduleroot/.travis.yml
+rvm: 2.3.1
+# Use anonymous, read-only interface to clone
+before_script: "echo 'git_base: https://github.com/' >> modulesync.yml"
+script: msync update -n katello --noop
```

Comment:
Credits to Dominic, he implemented it in theforeman. I just copied it and updated the namespace. I also wonder if the ```-n katello``` is even needed since there's a config file.